.github/workflows/ci.yml

```diff
@@ -128,41 +185,44 @@ jobs:
           max_attempts: 5
           retry_wait_seconds: 30
           command: |
-            composer update --optimize-autoloader --no-interaction --no-progress ${{ matrix.composer-flags }}
+            composer update --optimize-autoloader --no-interaction --no-progress --no-scripts ${{ matrix.composer-flags }}
```

Comment:
can you describe the change here?

Reply:
(i mean in file, not in github discussion)

Reply:
It's described in the PR:

> Fix CI warnings caused by installing dev-tools on `post-autoload-dump` hook. It did not affect main process, but also it wasn't needed at all, because dev-tools are only used locally and in SCA workflow, where are installed explicitly

When you install main project dependencies, dev-tools are installed automatically. But we don't need this behavior in CI where older versions of PHP are executed. I am not sure if I should add this kind of information there, inside a file.

Reply:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7366#discussion_r1374332740

I'm asking for this in file itself, so John Doe half year from now will not be wondering about this (and I find this optimization non-intuitive, related only to our repo specifics), nor look for PR that was originally introducing it.

Reply:
externalised to https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7403